projects/1.21/assets/evilcraft/evilcraft/lang/zh_cn.json

```
@@ -0,0 +1,764 @@
+{
+  "_comment": "General",
+  "itemGroup.evilcraft": "邪恶工艺",
+  "general.evilcraft.info.empty": "空",
+  "general.evilcraft.description": "Minecraft的一个邪恶模组。",
+  
+  "_comment": "Sound names",
+  "subtitles.evilcraft.effect.vengeancebeam.base": "复仇光束",
+  "subtitles.evilcraft.effect.vengeancebeam.start": "复仇光束：开始",
+  "subtitles.evilcraft.effect.vengeancebeam.stop": "复仇光束：停止",
+  "subtitles.evilcraft.mob.vengeancespirit.ambient": "复仇之魂：低吼",
+  "subtitles.evilcraft.mob.vengeancespirit.death": "复仇之魂：死亡",
+  "subtitles.evilcraft.effect.box.beam": "永恒灵魂箱光束",
+  "subtitles.evilcraft.effect.page.flipsingle": "翻页声",
+  "subtitles.evilcraft.effect.page.flipmultiple": "连续翻页声",
+  
+  "_comment": "Blocks",
+  "block.evilcraft.inner_block.info": "方块：%s",
+  "block.evilcraft.blood": "血液",
+  "block.evilcraft.poison": "毒液",
+  "block.evilcraft.blood_chest": "血能箱",
+  "block.evilcraft.blood_chest.info": "用血液修复物品。",
+  "block.evilcraft.blood_infuser": "血能注入器",
+  "block.evilcraft.dark_ore": "暗黑宝石矿石",
+  "block.evilcraft.dark_ore_deepslate": "深层暗黑宝石矿石",
+  "block.evilcraft.dark_block": "暗黑宝石块",
+  "block.evilcraft.blood_stain": "血迹",
+  "block.evilcraft.blood_stain.info": "生物摔死后产生的血迹。",
+  "block.evilcraft.lightning_bomb": "雷击炸弹",
+  "block.evilcraft.bloody_cobblestone": "沾满血的圆石",
```

```suggestion
  "block.evilcraft.bloody_cobblestone": "血石",
```
我认为此处应参 Mossy Cobblestone 苔石

---

参考的是1.18

---

怪事，我不知道从那比对出来了这个的差异。
不改了